### ISSUE TITLE
S42B 2.0 Closed Loop Not Working. 

### ISSUE THREAD

**Issue by rennwaffen** [bigtreetech/BIGTREETECH-S42B-V1.0#35]
Just installed the new S42B 2.0, and closed loop doesn't seem to want to work. I'm waiting on a response from BTT (which takes a little while) so I figured I'd ask here in the meantime. I also have a V1.1 on my X axis that works flawlessly. Any idea as to what the issue could be? 
Info:
Calibration went fine (Supposedly)
DIP 3 is set to ON
DIP 1, 2, 4 are set to OFF
32 Microstepping
Tried currents as high as 1000mA (2000 setting) with no luck.
Board: SKR 1.4 Turbo

Notice the deg. value not changing as the axis moves:

![ezgif-3-3ef1ae54515c](https://user-images.githubusercontent.com/59668075/100808106-6384aa00-33e8-11eb-9d1f-824f88b25eb1.gif)

**Comment by dzid26** [comment]
Swap one one of the phase polarity or DirPin direction in the menu:
https://github.com/bigtreetech/BIGTREETECH-S42B-V1.0/issues/33#issue-740859328

**Comment by N6MQD** [comment]
There is another issue posted that may be your issue.  DIP 2 is for Closed Loop mode.  DIP 1 is for Calibration.  DIP 3 and 4 are for subdivision.

**Comment by marshr32** [comment]
I know this is from a while back, but for any other poor bastard looking for a solution, try this

https://github.com/bigtreetech/BIGTREETECH-S42B-V1.0/issues/24#issuecomment-950551658
